VERSION

```diff
@@ -0,0 +1 @@
+2.0.0-RC1
```

Comment:
we need to update it when we actually do a tag, then again straight after to `-dev`